tests/unit/core/test_processor.py

```diff
@@ -40,12 +40,13 @@ def test_valid_event_request_partitioned():
     )
 
 
-def test_invalid_event_request():
-    with pytest.raises(EmptyResultSetException) as e:
-        processor.process_event_request(
-            test_data.INVALID_EVENT_QUERY_INVALID_STOP_DATE
-        )
-    assert str(e.value) == "Empty result set"
+def test_event_request_causing_empty_result():
```

Comment:
:100:

Reply:
Second look: Should the invalid event query return a 400_BAD_REQUEST or is an empty table ok?